mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/NioDatagramWorker.java

```diff
@@ -55,7 +55,7 @@
  * A class responsible for registering channels with {@link Selector}.
  * It also implements the {@link Selector} loop.
  */
-public class NioDatagramWorker extends AbstractNioWorker {
+public class NioDatagramWorker extends AbstractNioWorker implements Boss {
```

Comment:
Currently NioDatagramWorker is used for server channel and NioWorker is used for client channel and child channels. It also extends AbstractNioWorker. 

So may be:
1. rename NioDatagramWorker to NioDatagramBoss
2. Also, NioDatagramBoss not extend AbstractNioWorker
3. I think that would mean we need NioServerDatagramChannel which takes NioDatagramBoss as worker